hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/OMClientRequest.java

```diff
@@ -122,6 +123,13 @@ public OMRequest preExecute(OzoneManager ozoneManager)
     return omRequest;
   }
 
+  public OmLockOpr.OmLockInfo lock(OzoneManager ozoneManager, OmLockOpr lockOpr) throws IOException {
+    return null;
+  }
+
+  public void unlock(OmLockOpr lockOpr, OmLockOpr.OmLockInfo lockInfo) {
+  }
```

Comment:
What is the purpose of defining these stubs in the parent class? I don't see why anyone would call them from outside (public) or using the generic `OMClientRequest` type.